cli/defenseclaw/commands/cmd_upgrade.py

```
@@ -1822,6 +1854,345 @@ def _fetch_latest_version() -> str | None:
+        resolver_args.extend(("--version", intent["target_version"]))
+
+    click.echo(f"  {ux.dim('→')} Authenticating the DefenseClaw {resolver_version} release-owned upgrade resolver ...")
+    try:
+        with _authenticated_release_resolver(resolver_version) as (bash, resolver):
+            bash.assert_stable()
+            child_env = os.environ.copy()
+            child_env.pop("VERSION", None)
+            child_env.pop("PYTHONHOME", None)
+            child_env.pop("PYTHONPATH", None)
+            child_env[_UPGRADE_HANDOFF_ENV] = "1"
+            completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)